pkg/service/replay/replay.go

```diff
@@ -249,8 +248,8 @@ func (r *Replayer) Start(ctx context.Context) error {
 			}
 		}
 
-		if i < len(testSetIDs)-1 || r.config.Test.SkipCoverage {
-			err = HookImpl.AfterTestSetRun(ctx, testRunID, testSetID, models.TestCoverage{}, len(testSetIDs), testSetResult)
+		if i < len(testSets)-1 || r.config.Test.SkipCoverage {
```

Comment:
Let's say I have test-sets 1,2,3,4,5 I selected to run 4,5. In this case 4th test-set will not be uploaded